app/components/dialogs/AddSongDialog.tsx

```
@@ -0,0 +1,302 @@
+          value={songData.releasedate}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            setSongData({ ...songData, releasedate: e.target.value })}
+          className=" w-full"
+          title="Release Date"
+          label="Release Date"
+          leadingIcon={null}
+          trailingIcon={null}
+        />
+
+        <div className="">
+          <label className="block text-sm font-medium mb-2">Audio File</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)